src/arrayMethodSort.js

```diff
@@ -4,8 +4,38 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  const compareStr = (a, b) => {
+    const strA = String(a);
+    const strB = String(b);
+
+    if (strA > strB) {
+      return 1;
+    } else if (strA === strB) {
+      return 0;
+    } else {
+      return -1;
+    }
```

Comment:
No need in else statement
```suggestion
    if (strA > strB) {
      return 1;
    }
    
    if (strA === strB) {
      return 0;
    }
    
    return -1;
```